New high pop station - Serpentcrest

TytosB opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->

i have created a new high pop station, 'serpentcrest' based around the gimmick of a syndicate station recently conquered by Nanotrasen. 

here's some lore if you're interested.

"Serpentcrest station was originally founded as a mining outpost owned by a Gorlex subsidiary in order to 
extract the very rich gold and silver deposits present. however, as the borders shifted in the never ending 
proxy war between the syndicate and NT, its position became more strategically valuable than its ore, and
it was moved to direct Gorlex administration and converted into a forward outpost for launching and supplying operations into NT space. the structure was massively expanded, with new fully man-made additions to the station expanding out from the original asteroid base in all directions. the syndicate, being ever fond of iconography to an obnoxious degree, built
the station to literally take the shape of a snake curling around the gold-rich rock.

Very recently, the station came under NT control after a very violent and prolonged battle. the station 
still bears many unmistakable marks of its history as a syndicate base, as well as many scars of the recent
battle, and the space around it is now littered with the wrecks of various NT and syndicate warships."

the station also has its own cargo shuttle and evac shuttle that go with the theme. as well as two small ruined ships nearby it.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->

because i think it would be a very interesting and unique station in the pool with a very special concept and story.

## Technical details
<!-- Summary of code changes for easier review. -->

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

<img width="5984" height="5600" alt="Serpentcrest-0" src="https://github.com/user-attachments/assets/cd4f3f55-25bc-47cc-8745-beff0f256783" />
<img width="544" height="224" alt="Serpentcrest-1" src="https://github.com/user-attachments/assets/1424c9e9-12d4-4f15-938c-103302ffeba3" />
<img width="768" height="416" alt="Serpentcrest-2" src="https://github.com/user-attachments/assets/4f673bc5-ef81-4b22-beba-062ccb257059" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl:
- add: added new station: Serpentcrest. along with accompanying evac and cargo shuttles

Lukasz825700516 commented:
I like the concept, but:
- most of rooms seem to me to be way too empty compared to their size
- TEG seems to have no radiator setup
- very long maintenance corridors feel strange to me
- cargo has no outer reinforced walls
- medical has no outer reinforced walls
- head of personel is so far from the general service area
- clean chapel tiles clash so much with the dirty ones it hurts me

TytosB commented:
> I like the concept, but:
> 
>     * most of rooms seem to me to be way too empty compared to their size
> 
>     * TEG seems to have no radiator setup
> 
>     * very long maintenance corridors feel strange to me
> 
>     * cargo has no outer reinforced walls
> 
>     * medical has no outer reinforced walls
> 
>     * head of personel is so far from the general service area
> 
>     * clean chapel tiles clash so much with the dirty ones it hurts me

thanks for the response, however, i made cargo and med have normal walls as thats pretty standard on most maps [box,bagel,marathon, fland]

Minemoder5000 commented:
So just doing a quick skim over this, you'll need to unmap most if not all of the syndicate contraband. The vestine, magboots, eva suits and masks, syndiejuice, susboxes, and everything else syndicate contraband.

TytosB commented:
@alexalexmax to address a few of these, thats a nanomed restock not a chemvend restock, the emitters/tesla coil/ etc is all in flatpacks because that makes setting it up faster and easier, i forgor the cargo computer when i was remaking cargo, and while the syndicate microwaves are very common, there are about as many normal microwaves on the station as a normal map would have. rather, the syndie microwaves actually provide the advantage of letting more people have the ability to cook food without making cooking ids too easy. also there are no less than four different airlocks in maints [counting the backup maints dock as one] for people to get into space, in addition to evac, arrivals, the shuttle bay construction bay, cargo, and 6 escape pods. so im really not sure how i could add any more ways to get into space.

TeenSarlacc commented:
> It's why I said that they should remove components to make the item non-functional.

I missed that :clueless:

TytosB commented:
@ArtisticRoomba 

ive fixed most of the issues listed, still working on engineering power setup and the main plaza area, but there are a few i wanted to comment on

"PA control computer facing the wrong direction" it actually isnt, once you add the lv wire to complete it, it corrects its position, that was the only rotation that made the pa system work, all others it wont scan parts because its not lined up correctly.

"Maints chem having a full dispenser, chemvend board and hotplate is iffy." the dispenser is empty and its a chem master 4000 board, do you still want me to remove them?

"I wouldn't give security a webvest and C-20R" i removed the gun component from the c20 since you gave me the idea with the n1984, neither are meant to really be used and are more for storytelling

"Stuff like vents and scrubbers should not be hidden under objects, they should be visible." i fixed most of these, but the ones in sci directly under the lathes are on purpose, as it helps clear the heat generated by upgraded lathes faster.

"Chem should only get one hotplate" i cant see a good reason for this, its just a basic QOL thing like giving them roundstart plasma, except this doesnt even remove interdepartment interaction of having to ask for plasma.

as for the plaza, i have a few ideas, i could just move the eva room south and turn the area where it is now into more maints. alternatively, i might be able to figure out a way to squeeze a courtroom into the upper park area instead, as the station doesnt have one.

TytosB commented:
what do we think, better or worse?
<img width="1108" height="760" alt="court" src="https://github.com/user-attachments/assets/a7dd0103-a372-4b1d-aed7-cb1007d891b7" />

TytosB commented:
<img width="946" height="663" alt="fwwef" src="https://github.com/user-attachments/assets/2373a1ef-d775-46c8-a27f-d63b71e6326e" />

alternatively

TytosB commented:
i think i got all the issues raised in one way or another

TytosB commented:
im adding some custom beacons since for some reason when i try to custom name blank beacons they just refuse to show up on maps in postinit

TytosB commented:
> Here are some things I found while playtesting this map on Leviathan:
> 
>     * Decals can be wiped away with space cleaner, unsure if intentional or not.
> 
>     * AME doesn't spawn with any AME fuel.
> 
>     * The kitchen freezer has the wrong air vent. It should be a passive one to allow the air to be cooled.
> 
>     * Related to the kitchen freezer, it doesn't have any air stoppers. I don't know if that's intended or not with current mapping standards.
> 
>     * The Cargo shuttle was unable to be moved in testing without placing a new shuttle console.
> 
>     * Moving a TEG mix from the main Atmos room to the TEG room is difficult due to the limited space available for pipes.
> 
>     * The evac shuttle needs `fixgridatmos` run on it.

thanks ress, can you tell me which decal in particular was cleanable? some are meant to be and some arent

Kowlin commented:
> > Here are some things I found while playtesting this map on Leviathan:
> > ```
> > * Decals can be wiped away with space cleaner, unsure if intentional or not.
> > 
> > * AME doesn't spawn with any AME fuel.
> > 
> > * The kitchen freezer has the wrong air vent. It should be a passive one to allow the air to be cooled.
> > 
> > * Related to the kitchen freezer, it doesn't have any air stoppers. I don't know if that's intended or not with current mapping standards.
> > 
> > * The Cargo shuttle was unable to be moved in testing without placing a new shuttle console.
> > 
> > * Moving a TEG mix from the main Atmos room to the TEG room is difficult due to the limited space available for pipes.
> > 
> > * The evac shuttle needs `fixgridatmos` run on it.
> > ```
> 
> thanks ress, can you tell me which decal in particular was cleanable? some are meant to be and some arent

It was the ones regarding syndicate logos I'm pretty sure.

Kowlin reviewed (commented):
Round 2 of play testing, whew, that was a VERY long round!

Atmos:
 - TEG hot loop end has one of the volumetric pumps going the wrong way.

AI:
 - Camera vision is lacking, feels very claustrophobic. No vision into space.
 - No freezer camera.
 - AI core looks a bit too easy to disconnect. (and make into a shuttle)

Borgs:
 - There is a lack of chargers on a very big map, it would use some chargers that are not directly in Science.

Disposals:
 - When trash is deposited into disposals and flushed to recycling, and recycling is off, it becomes difficult to easily recycle all materials. This is because there is no easy path back for trash when it takes the turn.

Security:
 - Warden's fax is easily accessible and steal able.
 - There is no prisoner outfit locker in/around the GenPop locker room.

Captain's office:
 - Lacks a communication's console.

Power & Cabling:
 - AI Sat needs a SMES bank in order to be a one way power grid.
 - The single generator in the AI sat is insufficient to fully power the sat alone.
 - Station power is easy to sabotage because there is only a singular HV wire going through out the station, a second line may have to be added for redundancy.

TytosB commented:
>     * Station power is easy to sabotage because there is only a singular HV wire going through out the station, a second line may have to be added for redundancy.

i hope the engis got to have a nice long talk with bradley
